osquery/logger/logger.cpp

```diff
@@ -480,7 +440,7 @@ const std::vector<std::string>& BufferedLogSink::enabledPlugins() const {
 }
 
 BufferedLogSink::~BufferedLogSink() {
-  disable();
```

Comment:
We don't know if google:: objects are already destructed. The best thing we can do is just to stop messages from being processed in send.

Reply:
I might not be true here. @akindyakov your feedback will be appreciated